std/range/package.d

```diff
@@ -404,18 +404,18 @@ Iterates range $(D r) with stride $(D n). If the range is a
 random-access range, moves by indexing into the range; otherwise,
 moves by successive calls to $(D popFront). Applying stride twice to
 the same range results in a stride with a step that is the
-product of the two applications.
-
-Throws: $(D Exception) if $(D n == 0).
+product of the two applications. It is an error for $(D n) to be 0.
```

Comment:
Not a huge fan of such wording. I'd have expected a "$(D n) must not be empty" or "$(D n) shall not be empty". But whatevs.